### DISCUSSION TITLE
Why the maximum size does not support 512px

### DISCUSSION THREAD

**Discussion by nexmoe** [mat/besticon#66] [Q&A]
When I set the parameter max to 512, he returns ` {"error":"bad size parameter"}`

**Comment by jtayler** [comment]
I find a lot of simple WordPress sites seem to only have 512 e.g. <meta property="og:image" content="whateverdomain.com/wp-content/uploads/2020/03/cropped-favicon.jpg"/> which really SHOULD work but I guess are ignored because I can ask for 500px but not 512px?
